frontend/src/scenes/settings/SettingsMap.tsx

```diff
@@ -614,6 +614,14 @@ export const SETTINGS_MAP: SettingSection[] = [
             },
         ],
     },
+    {
+        level: 'project',
```

Comment:
Technically, these are environment-level, so you should match the existing pattern until the environment code is removed.